hw/ip/otbn/doc/_index.md

```diff
@@ -47,17 +47,15 @@ which has been formally verified within the [Fiat Crypto project](http://adam.ch
 
 # Instruction Set
 
-OTBN is a processor with a custom instruction set, which is described in this section.
+OTBN is a processor with a custom instruction set.
+The full ISA description can be found in our [ISA manual](isa.md).
```

Comment:
you need to use a hugo link here, this one doesn't work in rendered form.

Reply:
Huh, I could have sworn I checked that. Apparently not! Thanks for spotting it.